app/src/main/res/drawable/ic_print.xml

```diff
@@ -0,0 +1,15 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="21.6dp"
+    android:viewportWidth="20"
+    android:viewportHeight="18">
```

Comment:
Where did you find the icon because its size is incorrect

Reply:
I got it from the SDK.